classes/scripts/runner.php

```diff
@@ -608,7 +606,7 @@ public static function enableAutorun($name)
 						{
 							$autorunner->writeError($exception->getMessage());
 
-							exit($exception->getCode());
+							exit(2);
```

Comment:
What about `exit($exception->getCode() + 2)`? It works only if `getCode` returns a non-negative integer.